test/VariablesTest.cs

```diff
@@ -42,6 +42,18 @@ public void AddSecretTest()
         }
 
         [Test]
+        public void CountVariablesTest()
+        {
+            string variableUri = $"test:///resources/{TestAccount}/{Constants.KIND_VARIABLE}";
+
+            ClearMocker();
+            Mocker.Mock(new Uri(variableUri + "?count=true&search=dummy"), @"{""count"":0}");
```

Comment:
I think that test with Count different from 0 and 1 will be more effective

Reply:
Fixed